src/internal/m365/collection/drive/collections_tree_test.go

```diff
@@ -349,72 +365,19 @@ func (suite *CollectionsTreeUnitSuite) TestCollections_MakeMetadataCollections()
 	}
 }
 
-// TODO(keepers): implement tree version of populateDriveCollections tests
-
-// TODO(keepers): implement tree version of TestGet single-drive tests
-
-func (suite *CollectionsTreeUnitSuite) TestCollections_MakeDriveCollections() {
-	drive1 := models.NewDrive()
-	drive1.SetId(ptr.To(idx(drive, 1)))
-	drive1.SetName(ptr.To(namex(drive, 1)))
-
-	table := []struct {
-		name         string
-		c            *Collections
-		drive        models.Driveable
-		prevPaths    map[string]string
-		expectErr    require.ErrorAssertionFunc
-		expectCounts countTD.Expected
-	}{
-		{
-			name:      "not yet implemented",
-			c:         collWithMBH(mock.DefaultOneDriveBH(user)),
-			drive:     drive1,
-			expectErr: require.Error,
-			expectCounts: countTD.Expected{
-				count.PrevPaths: 0,
-			},
-		},
-	}
-	for _, test := range table {
-		suite.Run(test.name, func() {
-			t := suite.T()
-
-			ctx, flush := tester.NewContext(t)
-			defer flush()
-
-			colls, paths, delta, err := test.c.makeDriveCollections(
-				ctx,
-				test.drive,
-				test.prevPaths,
-				test.c.counter,
-				fault.New(true))
-
-			// TODO(keepers): awaiting implementation
-			test.expectErr(t, err, clues.ToCore(err))
-			assert.Empty(t, colls)
-			assert.Empty(t, paths)
-			assert.Empty(t, delta.URL)
-
-			test.expectCounts.Compare(t, test.c.counter)
-		})
-	}
-}
-
-// TODO(keepers): implement tree version of TestGet multi-drive tests
-
+// This test is primarily aimed at multi-drive handling.
+// More complicated single-drive tests can be found in _MakeDriveCollections.
 func (suite *CollectionsTreeUnitSuite) TestCollections_GetTree() {
-	metadataPath, err := path.BuildMetadata(
-		tenant,
-		user,
-		path.OneDriveService,
-		path.FilesCategory,
-		false)
-	require.NoError(suite.T(), err, "making metadata path", clues.ToCore(err))
-
-	drive1 := models.NewDrive()
-	drive1.SetId(ptr.To(idx(drive, 1)))
-	drive1.SetName(ptr.To(namex(drive, 1)))
+	// metadataPath, err := path.BuildMetadata(
+	// 	tenant,
+	// 	user,
+	// 	path.OneDriveService,
+	// 	path.FilesCategory,
+	// 	false)
+	// require.NoError(suite.T(), err, "making metadata path", clues.ToCore(err))
```

Comment:
nit: is this a stale comment that can be removed?

Reply:
it'll come back later.  I have more assertions than necessary for the test atm, because it's not a fully implemented test.  Was easier to comment things out than to fix the assertions.